src/web/mjs/connectors/ShueishaMangaPlus.mjs

```diff
@@ -14,14 +14,33 @@ export default class ShueishaMangaPlus extends Publus {
         this.rootType = 'MangaPlus.Response';
     }
 
+    _getLanguage(language) {
+        switch (language) {
```

Comment:
It is recommend to use a map instead of switch case when possible, e.g.
```javascript
function _getLanguages(language) {
    const languages = {
        1: '[es]', 2: '[fr]', // ...
    };
    return languages[language] || '[en]';
}

```

Reply:
Thanks for the feedback. I changed the code.